src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/RollbacksSpec.groovy

```diff
@@ -76,7 +76,7 @@ and at least 1 path must remain safe"
         then: "System fails to install rules on desired path and tries to retry path installation"
         Wrappers.wait(rerouteDelay + WAIT_OFFSET, 0.1) {
             assert northbound.getFlowHistory(flow.flowId).find {
-                it.action == "Flow rerouting" && it.details.matches(/Reason: .*?: retry #1/)
+                it.action == "Flow rerouting" && it.taskId =~ (/[^a-z_A-Z0-9-\s]* : retry #1 :/)
```

Comment:
Is this expected? 'retry #' really made sense in the details description, not sure about task id.

Reply:
As for me "retry #" in task id section is useful to track reroute history in logs. Actually we can duplicate "retry #" to reroute request reason.

Reply:
I don't have a strong opinion here, just checking that this is not a typo